hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1143,6 +1145,64 @@ public function upgrade_1032() {
     return TRUE;
   }
 
+  /**
+   * Removes the "Pension Type" item from the
+   *  "Administer -> Customize Data and Screens -> Dropdowns" menu
+   *
+   * The option group this menu item links to has been removed by PCHR-1820,
+   * but the menu item itself wasn't, so we're deleting it now.
+   *
+   * @return bool
+   */
+  public function upgrade_1033() {
+    civicrm_api3('Navigation', 'get', [
+      'sequential' => 1,
+      'name' => 'hrjc_pension_type',
+      'url' => ['LIKE' => 'civicrm/admin/options?gid%'],
+      'api.Navigation.delete' => ['id' => '$value.id'],
+    ]);
+    CRM_Core_BAO_Navigation::resetNavigation();
+
+    return TRUE;
+  }
+
+  /**
+   * Update the URLs of all menu items pointing to Job Contracts options under
+   * the "Administer -> Customize Data and Screens -> Dropdowns" menu to have
+   * the option group name instead of its ID
+   *
+   * @return bool
+   */
+  public function upgrade_1034() {
+    $dropdownMenuItems = [
+      'hrjc_contract_type',
+      'hrjc_location',
+      'hrjc_pay_cycle',
+      'hrjc_benefit_name',
+      'hrjc_benefit_type',
+      'hrjc_deduction_name',
+      'hrjc_deduction_type',
+      'hrjc_revision_change_reason',
+      'hrjc_contract_end_reason',
+    ];
+
+    foreach($dropdownMenuItems as $menuItem) {
+      civicrm_api3('Navigation', 'get', [
+        'sequential' => 1,
+        'name' => $menuItem,
+        'url' => ['LIKE' => 'civicrm/admin/options?gid%'],
+        'api.Navigation.create' => [
+          'id' => '$value.id',
+          'url' => "civicrm/admin/options/{$menuItem}?reset=1"
```

Comment:
Personal preference, but I like `sprintf` more because of its type safety and that it avoids potentially evaluating something you don't want to.